### PR TITLE
Add stacktrace logging

### PR DESCRIPTION
**Pull request by ruslan-bikkinin**
This PR add stacktrace logging which is will be very handy to debug errors.

### CODE REVIEW

File: www/codePushUtil.ts [hunk 1]
```diff
@@ -66,7 +66,11 @@ class CodePushUtil {
      */
     public static logError(message: String, error?: Error): void {
         var errorMessage = message || "" + " " + CodePushUtil.getErrorMessage(error);
-        console.error(CodePushUtil.TAG + " " + errorMessage);
+        var stackTrace = '';
```
Comment (sergey-akhalkov) [inline www/codePushUtil.ts]: It would be great to use `let` instead of `var` and `const` instead of `let` if possible.

File: www/codePushUtil.ts [hunk 2]
```diff
@@ -66,7 +66,11 @@ class CodePushUtil {
...
+        if (error.stack) {
+            stackTrace += '. Stacktrace: ' + error.stack;
+        }
+        console.error(CodePushUtil.TAG + " " + errorMessage + stackTrace);
```
Comment (sergey-akhalkov) [inline www/codePushUtil.ts]: Let's use `template literals` for text formatting instead of `+`

File: bin/www/codePushUtil.js [hunk 3]
```diff
@@ -36,7 +36,11 @@ var CodePushUtil = (function () {
     };
     CodePushUtil.logError = function (message, error) {
         var errorMessage = message || "" + " " + CodePushUtil.getErrorMessage(error);
-        console.error(CodePushUtil.TAG + " " + errorMessage);
+        var stackTrace = '';
+        if (error.stack) {
+            stackTrace += '. Stacktrace: ' + error.stack;
```
Comment (sergey-akhalkov) [inline bin/www/codePushUtil.js]: Stacktrace -> StackTrace

File: www/codePushUtil.ts (hunk 1 above)
Comment (ruslan-bikkinin) [reply www/codePushUtil.ts]: Done, thx

File: www/codePushUtil.ts (hunk 2 above)
Comment (ruslan-bikkinin) [reply www/codePushUtil.ts]: Done, thx

File: bin/www/codePushUtil.js (hunk 3 above)
Comment (ruslan-bikkinin) [reply bin/www/codePushUtil.js]: Done, thx

**Pull request merged**